r/run_model.R

```diff
@@ -0,0 +1,25 @@
+library(optparse)
+library(readr)
+library(tidyr)
+
+# Create a CLI ------------------------------------
+# Two args: `input_dir`, `output_dir`
+option_list <- list(
+  make_option("--input_dir", type="character", default="/input", metavar="character"),
```

Comment:
It's a personal preference but I prefer to use dashes `-` as word separator because it's only requires to press one key instead of two on a US keyboard (better accessibility). `--input_dir` would then become `--input-dir`

```suggestion
  make_option("--input-dir", type="character", default="/input", metavar="character"),
```